qid-batch-runner.yml

```diff
@@ -140,4 +147,6 @@ spec:
               - key: our-public-key
                 path: "our_public_key.asc"
               - key: qm-supplier-public-key
-                path: "other_public_key.asc"
+                path: "supplier_QM_dummy_public.asc"
+              - key: ppo-supplier-public-key
+                path: "supplier_ppo_dummy_public.asc"
```

Comment:
`dummy_ppo_supplier_public_key.asc` should be renamed to `supplier_PPO_dummy_public.asc`